_authors/osci_2023/vishnu2411.markdown

```diff
@@ -0,0 +1,14 @@
+---
+name: T Vishnuvardhan Reddy
+short_name: Vishnu
+twitter: iamvishnuv7
+github: vishnu2411
+
```

Comment:
nit: can the newline be removed here and inserted after the next line `---` so that there is a newline between the top section and the introduction below.

Please start your intro with `**<Name>** is a passionate software developer...`

i.e. `**T Vishnuvardhan Reddy** is a passionate software developer...`